packages/core/src/browser/theming.ts

```diff
@@ -108,10 +111,10 @@ export class ThemeService {
     }
 
     /**
-     * The default them. If that is not applicable, returns with the fallback theme.
+     * The default theme. If that is not applicable, returns with the fallback theme.
      */
     get defaultTheme(): Theme {
-        return this.themes[this._defaultTheme || this.fallbackTheme] || this.themes[this.fallbackTheme];
+        return this.themes[this._defaultTheme || this.fallbackTheme];
```

Comment:
Why did you remove it?

Reply:
Because while fixing the typo in the comment, it seemed to me to be useless ?

If `this._defaultTheme` is empty/undefined, *it will* use `this.fallbackTheme`.

So the expression will already be equivalent to `this.themes[this._defaultTheme]`, so writing it a second time felt weird to me.

Reply:
If `this._defaultTheme` is defined `"foo"` but there is no such theme we have to fall back to the fallback. Please revert.

Reply:
thanks

Reply:
No. https://github.com/theia-ide/theia/issues/2391

Reply:
Also, I feel like the `fallbackTheme` is some kind of `defaultDefaultTheme` to the `_defaultTheme` member of the ThemeService object.

Is `FrontendApplicationConfigProvider` meant to be used by users or by developers ?

Because I would only rely on that, and get rid of the `fallbackTheme` and make it as a default to the frontend app config provider. Throw an error if default theme is not found.

Reply:
It doesn't change the fact that users shouldn't touch the `package.json`, and setting an unknown theme as the default application theme is a mistake. Still not worth an error?

Reply:
No.

Reply:
Thanks for the feedback, but how about a `console.warn('defaulttheme not found')`?

Reply:
It's up to you.